.gitignore

```diff
@@ -11,3 +11,5 @@ MANIFEST
 ninetofiver_venv/
 docs/_build/
 /media/
+/Import_scripts/
```

Comment:
What framework/coding convention required the "Import_scripts" folder? If this is something only you are using, please do not add it to the public .gitignore file.